core/src/main/java/jenkins/triggers/SCMTriggerItem.java

```diff
@@ -116,6 +120,11 @@
                 return delegate.asProject().scheduleBuild2(quietPeriod, null, actions);
             }
             @Override public PollingResult poll(TaskListener listener) {
+                SCMDecisionHandler veto = SCMDecisionHandler.firstShouldPollVeto(asItem());
+                if (!veto.shouldPoll(asItem())) {
```

Comment:
@jtnord Thanks. It seems the commit comment bot got broken by the JIRA update, so there are no closings nor commit references…